src/main/clojure/clara/rules/compiler.clj

```diff
@@ -64,7 +64,13 @@
                         ;; Map of queries to the nodes hosting them.
                         query-nodes :- {sc/Any QueryNode}
                         ;; Map of id to one of the beta nodes (join, accumulate, etc).
-                        id-to-node :- {sc/Num BetaNode}])
+                        id-to-node :- {sc/Num BetaNode}
+                        ;; Function for sorting activation groups of rules for firing.
+                        activation-group-sort-fn
+                        ;; Function that takes a rule and returns its activation group.
+                        activation-group-fn
+                        ;; Function that takes facts and determines what alpha nodes they match.
```

Comment:
We're talking about the get-alphas-fn correct?  That takes multiple facts.

https://github.com/cerner/clara-rules/blob/0.13.0-RC4/src/main/clojure/clara/rules/compiler.clj#L1554